libs/domain/picking/src/components/filter-button/filter-button.component.ts

```diff
@@ -0,0 +1,43 @@
+import { resolve } from '@spryker-oryx/di';
+import { PickingListService } from '@spryker-oryx/picking';
+import { i18n, Size, subscribe } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { query } from 'lit/decorators.js';
+import { tap } from 'rxjs';
+
+export class FilterButtonComponent extends LitElement {
+  @query('oryx-picking-filters') protected filters?: HTMLElement;
+  @query('input') protected input?: HTMLInputElement;
+
+  @subscribe()
+  protected selectedFilter = resolve(PickingListService)
+    .getSortingQualifier()
+    .pipe(
+      tap((sortingQualifier) => {
+        if (!this.input) return;
+        this.input.checked = !!sortingQualifier;
+      })
+    );
```

Comment:
Can we change to:

```ts
protected selected = signal(
  resolve(PickingListService).getSortingQualifier()
);

<input ?checked=${!!this.selected()} />
```

Nitpick, can we move onclick below render?

Reply:
Unfortunately - not. Checked attr just and initial value, it does not have affect on the property after.

Reply:
switched to signals

Reply:
I've asked @dunqan to double check as this is expected to work.

Reply:
Don't you mind if I fix (or not) this later, to not to block this pr?

Reply:
Discussed with Stan. We justified the signal's implementation, but as I mentioned before this does not affecting the property, that why I added mixed solution that triggers the property and attribute